src/common/aes/aes_local.h

```diff
@@ -9,15 +9,15 @@ void oqs_aes128_ecb_enc_sch_ni(const uint8_t *plaintext, const size_t plaintext_
 
 void oqs_aes128_load_schedule_c(const uint8_t *key, void **_schedule);
 void oqs_aes128_free_schedule_c(void *schedule);
-void oqs_aes128_enc_sch_block_c(const uint8_t *plaintext, const void *_schedule, uint8_t *ciphertext);
 void oqs_aes128_ecb_enc_sch_c(const uint8_t *plaintext, const size_t plaintext_len, const void *schedule, uint8_t *ciphertext);
```

Comment:
Remove the `const` from `const size_t plaintext_len`.

Reply:
We have const size_t in a bunch of other places, which is technically unnecessary but I'll keep for now for consistency.